ui-admin/src/study/participants/participantList/ParticipantList.test.tsx

```diff
@@ -29,6 +31,38 @@ test('renders a participant with link', async () => {
   expect(participantLink).toHaveAttribute('href', `/${studyEnvContext.currentEnvPath}/participants/JOSALK`)
 })
 
+test('renders filters for participant columns', async () => {
+  const studyEnvContext = mockStudyEnvContext()
+  const { RoutedComponent } = setupRouterTest(<ParticipantList studyEnvContext={studyEnvContext}/>)
+  render(RoutedComponent)
+
+  //Assert that all 3 default columns have filter inputs
+  await waitFor(() => {
+    expect(screen.getAllByPlaceholderText('Search...')).toHaveLength(3)
+  })
```

Comment:
I think this could probably be simplified to use `findBy...` instead of `waitFor... getBy...`.

```suggestion
  const searchInputs = await screen.findByPlaceholderText('Search...')
  expect(searchInputs).toHaveLength(3)
```

https://testing-library.com/docs/queries/about/#types-of-queries

Reply:
Simplified in a281500b